packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -122,44 +77,7 @@ const CommonFinders find = CommonFinders._();
 typedef EvaluatorFunction = dynamic Function();
 
 /// Drives a Flutter Application running in another process.
-class FlutterDriver {
-  /// Creates a driver that uses a connection provided by the given
-  /// [serviceClient], [_peer] and [appIsolate].
-  @visibleForTesting
-  FlutterDriver.connectedTo(
```

Comment:
Instead of removing this constructor, make it a redirecting factory constructor.